docker-compose.yml

```diff
@@ -4,8 +4,6 @@ services:
   chowchow-db:
     image: postgres:latest
     container_name: chowchow-db
-    volumes:
```

Comment:
Can we keep the volume now that it's fixed on local ?

Reply:
I guess we can 😉